docs/content/contributing/terraform/review.md

```diff
@@ -0,0 +1,75 @@
+---
+title: AVM Review of Terraform Modules
```

Comment:
```suggestion
title: Initial Review of Terraform Modules
```

Reply:
Would like to keep the original title, we have plans to further split this into two sections namely "New Module Reviews" and "Existing Module Reviews" in future